tests/configs/docker_registry/kubetools.yml

```diff
@@ -0,0 +1,24 @@
+name: generic-app
+
+
+containerContexts:
+  generic-context:
+    image: generic-registry/generic-image
+    command: [generic-command]
+    ports:
+      - 80
+
+
+deployments:
+  generic-app:
+    containers:
+      with-default-registry:
+        command: [uwsgi, --ini, /etc/uwsgi.conf]
+        containerContext: generic-context
+
+      without-default-registry:
+        image: generic-image
+        command: [uwsgi, --ini, /etc/uwsgi.conf]
+        ports:
+          - 80
```

Comment:
Needs some renaming here:
* `with-default-registry` has a non-default registry in the containerContext
* `without-default-registry` doesn't have a registry, hence uses the default registry

Also worth changing the test to avoid mixing 2 things: one is using a containerContext, the other is not. This is not what we're testing here. So either there needs to be 4 cases (with/without registry/context) or only 2 that are identical except for the with/without registry aspect

Reply:
https://github.com/EDITD/kubetools/pull/141/commits/b69af2413f6bf45f793c6462ce127b2478927dd2